templates/blog/post_list.html

```diff
@@ -0,0 +1,23 @@
+{% extends 'base.html' %}
+
+{% block title %}
+  <title>Blog Posts</title>
+{% endblock %}
+
+{% block content %}
+  <h2>Posts</h2>
+  <div>
+    <ul>
+      {% for post in post_list %}
+        <li>
+          <h3><a href="{% url 'blog:post-detail' pk=post.id %}">{{ post.title }}</a></h3>
+          <div> Author: {{ post.owner }}</div>
+          <div>{{ post.created_time }}</div>
```

Comment:
It would be a good idea to format the date and time of the post creation in a more human-readable way. Django templates have a built-in filter for that, for example: `{{ post.created_time|date:"F j, Y, P" }}`. This will output the date as 'Month Day, Year, Time'.